app/api/blocks/route.ts

```diff
@@ -15,7 +17,7 @@ export async function GET(request: NextRequest) {
   try {
     const blocks = await fetchBlocksPaginated({
       pageIndex,
-      pageSize,
+      pageSize: Math.max(pageSize, DEFAULT_PAGE_SIZE),
```

Comment:
@pettinarip max? Wouldn't the point of the pagination to limit the pageSize if it's too big, via `min`?
ie. pageSize is 20, DEFAULT_PAGE_SIZE is 15, isn't the point to limit to 15?

Reply:
Seems to be working though so must be missing something

Reply:
Ah yes! good catch...should be `min`. Let me see